build-system/tasks/check-types.js

```diff
@@ -122,8 +170,11 @@ const TYPE_CHECK_TARGETS = {
   // introduced. It is okay to remove a file from this list only when fixing a
   // bug for cherry-pick.
   'pride': {
-    srcGlobs: PRIDE_FILES_GLOBS,
-    externGlobs: [CORE_EXTERNS_GLOB, 'build-system/externs/*.extern.js'],
+    srcGlobs: [...PRIDE_FILES_GLOBS],
+    externGlobs: [
+      ...globby.sync('build-system/externs/*.js'),
```

Comment:
Just provide the glob
```suggestion
      'build-system/externs/*.extern.js',
```